proposals/0005-webauthn.md

```
@@ -0,0 +1,292 @@
+- CBOR parsing
+- Attestation validation
+- Signature verification
+
+What Ktor will implement:
+
+- The idiomatic Kotlin DSL
+- HTTP request/response `kotlinx.serialization` (JSON handling)
+- Session/challenge state management
+- Populating the Ktor `AuthenticationContext`
+
+### Authentication Process Diagram
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)